mock/service/controller.go

```diff
@@ -73,7 +73,7 @@ func (s *service) CreateVolume(
 		case *csi.VolumeContentSource_Volume:
 			vid := req.GetVolumeContentSource().GetVolume().GetVolumeId()
 			// Check if the source volume exists.
-			if volID, _ := s.findVolNoLock("id", vid); volID > 0 {
+			if volID, _ := s.findVolNoLock("id", vid); volID >= 0 {
```

Comment:
This is an unrelated change. Can you move it to a separate PR?

Reply:
> @avalluri please disable "pagination should detect volumes added between pages and accept tokens when the last volume from a page is deleted" by replacing `It` with `XIt`, with a comment that links to #223 and [intel/pmem-csi#424 (comment)](https://github.com/intel/pmem-csi/pull/424#issuecomment-540499938).

Done.

Reply:
> This is an unrelated change. Can you move it to a separate PR?

Done: #227